content/api/trafiklab-apis/gtfs-sweden/realtime.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Realtime data
+aliases:
+  - "api/gtfs-regional-realtime"
+weight: 20
+date: 2019-06-10
+---
+
+The realtime GTFS Sweden data consists of data feeds describing disturbances, deviations, delays, and even realtime 
+GPS vehicle positions, separated per region or operator just like the GTFS Regional API (for performance reasons). 
+It should be combined with static data available in the GTFS Sweden API.
+
+In order to retrieve the data you need an API key. Technical details for fetching the data can be found in the
+API's OpenAPI specification. Trafiklabs GTFS documentation can help you to get started with GTFS files.
+
+{{% info %}}
+**Availability of regional data** differs per operator. See the [overview page](_index.md) to see which data is provided
+by the operator(s) you are interested in.
+{{% /info %}}
+
+{{% note %}}
+**Where to download**
+
+The dataset can be accessed through the following URLs:
+- [https://opendata.samtrafiken.se/gtfs-rt-sweden/{operator}/ServiceAlerts.pb?key={apikey}](https://opendata.samtrafiken.se/gtfs-rt-sweden/{operator}/ServiceAlerts.pb?key={apikey})
+- [https://opendata.samtrafiken.se/gtfs-rt-sweden/{operator}/TripUpdates.pb?key={apikey}](https://opendata.samtrafiken.se/gtfs-rt-sweden/{operator}/TripUpdates.pb?key={apikey})
+- [https://opendata.samtrafiken.se/gtfs-rt-sweden/{operator}/VehiclePositions.pb?key={apikey}](https://opendata.samtrafiken.se/gtfs-rt-sweden/{operator}/VehiclePositions.pb?key={apikey})
+
+Replace `{operator}` with the abbreviation of the operator you want to download. These abbreviations can be found in the
+OpenAPI specification, but are also listed on the [static data documentation page](static.md). Replace `{apikey}` with your own API key.
+{{% /note %}}
+
+{{% page-ref page="/docs/using-trafiklab/getting-api-keys" %}}
+
+## Available real-time data
+
+### ServiceAlerts
+
+Service alerts allow you to get information about disruptions on the transit network. This can be anything from planned
+roadworks (a certain stop might not get served for a few days) to electricity outages on a rail network. ServiceAlerts
+are broad and general information. Trafiklab's serviceAlerts are updated every 15 seconds.
+
+Delays and cancellations of individual trips are usually communicated
+using [Trip updates](#tripupdates).
+See [the table on the GTFS Regional overview page](./_index.md#which-operators-are-covered-by-this-dataset) to find out which
+operators are supported (Realtime data column).
+
+{{% info %}} ServiceAlerts can remain unchanged for relatively long periods (for example a couple of
+hours). You can prevent unnecessary downloads by making use
+of [Conditional HTTP requests](/docs/using-trafiklab-data/best-practices/conditional-get-requests.md). {{% 
+/info %}}
```

Comment:
Works, but would be more readable if the entire closing info tag was on one line